package.json

```diff
@@ -1,26 +1,37 @@
 {
   "name": "bookcovers",
   "version": "0.1.0",
+  "author": "e-e-e",
   "description": "A federated search for thumbnail images of book covers",
+  "engines": {
+    "node": ">=10"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/e-e-e/book-covers-api.git"
+  },
+  "homepage": "https://github.com/e-e-e/book-covers-api",
+  "bugs": {
+    "url": "https://github.com/e-e-e/book-covers-api/issues"
+  },
   "main": "src/index.js",
   "bin": {
     "bookcovers": "src/cli.js"
   },
   "dependencies": {
     "commander": "^3.0.2",
-    "eslint": "^6.4.0",
     "express": "^4.17.1",
```

Comment:
We can also probably remove express as it is currently only used by the example `server.js`